sql/expression/function/greatest_least_test.go

```diff
@@ -0,0 +1,183 @@
+package function
+
+import (
+	"github.com/stretchr/testify/require"
+	"gopkg.in/src-d/go-mysql-server.v0/sql"
+	"gopkg.in/src-d/go-mysql-server.v0/sql/expression"
+	"testing"
+)
+
+func TestGreatest(t *testing.T) {
+	testCases := []struct {
+		name string
+		args []sql.Expression
+		expected interface{}
+	}{
+		{
+			"null",
+			[]sql.Expression{
+				expression.NewLiteral(nil, sql.Null),
+				expression.NewLiteral(5, sql.Int64),
+				expression.NewLiteral(1, sql.Int64),
+			},
+			nil,
+		},
+		{
+			"negative and all ints",
+			[]sql.Expression{
+				expression.NewLiteral(int64(-1), sql.Int64),
+				expression.NewLiteral(int64(5), sql.Int64),
+				expression.NewLiteral(int64(1), sql.Int64),
+			},
+			int64(5),
+		},
+		{
+            "string mixed",
+            []sql.Expression{
+				expression.NewLiteral(string("9"), sql.Text),
+				expression.NewLiteral(int64(5), sql.Int64),
+				expression.NewLiteral(int64(1), sql.Int64),
+			},
+			float64(9),
+		},
+		{
+            "unconvertible string mixed ignored",
+            []sql.Expression{
+				expression.NewLiteral(string("10.5"), sql.Text),
+				expression.NewLiteral(string("foobar"), sql.Int64),
+				expression.NewLiteral(int64(5), sql.Int64),
+				expression.NewLiteral(int64(1), sql.Int64),
+			},
+			float64(10.5),
+		},
+		{
+            "float mixed",
```

Comment:
is this gofmt'd? The indentation seems off

Reply:
Ups, fixed.